plugins/catalog-backend-module-azure/src/lib/azure.ts

```diff
@@ -46,7 +46,18 @@ interface CodeSearchRequest {
   };
 }
 
-const isCloud = (host: string) => host === 'dev.azure.com';
+const isCloud = (host: string) => {
+  if (host === 'dev.azure.com') {
+    return true;
+  }
+
+  if (host === 'visualstudio.com' || host.endsWith('.visualstudio.com')) {
```

Comment:
Would this `host === 'visualstudio.com'` ever be `true`? My understanding it's always `{org}.visualstudio.com`. If that's right we should remove the first part of this check.